app/views/static/terms.html.erb

```diff
@@ -0,0 +1,162 @@
+<h1>Terms of Use for <%= t('sufia.product_name') %></h1>
```

Comment:
If you look at the individual commits, you'll see the diff.  I imported the original from Sufia, then reformatted, then made the necessary change, which was a hard-coded absolute URL path down near the bottom.  So, yes, we still need it.